Bottom: e2c5c3f1a96db43e8f88bd8473ef145d9f68f28f
Top:    f6a2e4123840c8381cf07c5afb32e9240d4493c2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 20:07:17 +0000

hid/gtk (GL): I think the polygon renderer works in mask mode now


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 0a709fc..a7b642c 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1405,11 +1405,8 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  common_fill_pcb_polygon (out->fgGC, &polygon, screen);
+  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
-  /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  hid_draw_fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   hid_draw_use_mask (&ghid_graphics, HID_MASK_OFF);
